editor/src/clj/editor/collision_object.clj

```diff
@@ -529,15 +530,15 @@
   (input project-settings g/Any)
 
   (property collision-shape resource/Resource
-            (value (gu/passthrough collision-shape-resource))
-            (set (fn [evaluation-context self old-value new-value]
-                   (project/resource-setter evaluation-context self old-value new-value
-                                            [:resource :collision-shape-resource]
-                                            [:build-targets :dep-build-targets])))
-            (dynamic edit-type (g/constantly {:type resource/Resource :ext #{"convexshape" "tilemap"}}))
-            (dynamic error (g/fnk [_node-id collision-shape]
-                                  (when collision-shape
-                                    (validation/prop-error :fatal _node-id :collision-shape validation/prop-resource-not-exists? collision-shape "Collision Shape")))))
+          (value (gu/passthrough collision-shape-resource))
+          (set (fn [evaluation-context self old-value new-value]
+                 (project/resource-setter evaluation-context self old-value new-value
+                                          [:resource :collision-shape-resource]
+                                          [:build-targets :dep-build-targets])))
+          (dynamic edit-type (g/constantly {:type resource/Resource :ext #{"convexshape" "tilemap"}}))
+          (dynamic error (g/fnk [_node-id collision-shape shapes]
+                                (or (validation/prop-error :fatal _node-id :collision-shape validation/prop-resource-not-exists? collision-shape "Collision Shape")
+                                    (validation/prop-error :fatal _node-id :collision-shape validation/prop-collision-shape-conflict? shapes collision-shape)))))
```

Comment:
Somehow the indentation got messed up from your edits.
```suggestion
            (value (gu/passthrough collision-shape-resource))
            (set (fn [evaluation-context self old-value new-value]
                   (project/resource-setter evaluation-context self old-value new-value
                                            [:resource :collision-shape-resource]
                                            [:build-targets :dep-build-targets])))
            (dynamic edit-type (g/constantly {:type resource/Resource :ext #{"convexshape" "tilemap"}}))
            (dynamic error (g/fnk [_node-id collision-shape shapes]
                             (or (validation/prop-error :fatal _node-id :collision-shape validation/prop-resource-not-exists? collision-shape "Collision Shape")
                                 (validation/prop-error :fatal _node-id :collision-shape validation/prop-collision-shape-conflict? shapes collision-shape)))))
```

Reply:
I have to set up IDEA if I want to continue doing small changes for the editor. This indentation issues getting annoying